CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # Changelog
 
+## [2.4.0]
+
+### Changed
+
+- Fixed bug with paths containing `.` and `..` as inputs to `uploadDirectory`.
+
```

Comment:
There isn't any new functionality in this release, so according to SemVer it should be a patch release, e.g. `2.3.1`.
See https://semver.org/

Not a blocking issue, of course, but it would help with the galloping version numbers.

Reply:
agreed, should be 2.3.1